VisaCheckout/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1 @@
+sdk=28
```

Comment:
shouldn't this be `30`?

Reply:
Yeah unfortunately testing API 28 and up requires us to build with Java 9 and we haven't found a good strategy for updating yet. There are a few related [GH issues](https://github.com/robolectric/robolectric/issues/5258) on the topic but we've deferred until we can get the actual build back to green. Once we migrate Visa Checkout into core we'll have a stable CI to investigate further.